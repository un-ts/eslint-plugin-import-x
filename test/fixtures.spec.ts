import path from 'node:path'

import { exec } from 'tinyexec'

describe('yarn pnp', () => {
  const yarnPnpDir = path.resolve('test/fixtures/yarn-pnp')
  const execOptions = { nodeOptions: { cwd: yarnPnpDir } }

  it('should just work', async () => {
    await exec('yarn', ['install', '--immutable'], execOptions)
    const result = await exec('yarn', ['lint'], execOptions)
    expect(result).toMatchSnapshot()
  }, 10_000)
})
