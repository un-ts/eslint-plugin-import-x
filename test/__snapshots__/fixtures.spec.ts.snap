// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`yarn pnp should just work 1`] = `
{
  "exitCode": 0,
  "stderr": "",
  "stdout": "",
}
`;
